components/esp-tls/esp_tls_wolfssl.c

```diff
@@ -310,6 +317,23 @@ static esp_err_t set_client_config(const char *hostname, size_t hostlen, esp_tls
 #endif /* CONFIG_WOLFSSL_HAVE_ALPN */
     }
 
+#ifdef CONFIG_WOLFSSL_HAVE_OCSP
+    /* increase error verbosity */
+    wolfSSL_CTX_EnableOCSP((WOLFSSL_CTX *)tls->priv_ctx, WOLFSSL_OCSP_CHECKALL);
```

Comment:
Can you please add return check for this API as well.
I see in the wolfSSL manual that it is of int return type.
```
WOLFSSL_CTX* ctx = wolfSSL_CTX_new( method ); int options; // initialize to option constant
...
int ret = wolfSSL_CTX_EnableOCSP(ctx, options); if(ret != SSL_SUCCESS){
    // OCSP is not enabled
}
```